WooCommerce/Classes/ViewRelated/Products/Edit Product/Edit Tags/ProductTagsViewController.swift

```diff
@@ -380,30 +380,26 @@ extension ProductTagsViewController: UITextViewDelegate {
             range.length == 1 && text == "", // Deleting last character
             range.location > 0, // Not at the beginning
             range.location + range.length == original.length, // At the end
-            original.substring(with: NSRange(location: range.location - 1, length: 1)) == "," // Previous is a comma
-        {
+            original.substring(with: NSRange(location: range.location - 1, length: 1)) == "," { // Previous is a comma
```

Comment:
This and all the other changes to this file were made via `bundle exec rake autocorrect`